### PR TITLE
Specify what is duplicate client in err msg and ug

### LINKED ISSUE #183: [PE-D] Write in UG (and add/edit client cmd) what is the condition of a duplicate client
There is no consistent rules for adding new clients, in terms of what is considered a duplicate client.

After typing "cadd n/Jane Doe p/91234567 e/jane@gmail.com a/Random Place", the client is added:

Typing the following obtains an error:
cadd n/JaneDoe p/91234567 e/jane@gmail.com a/Random Place
cadd n/Jane Doe p/1234567 e/jane@gmail.com a/Random Place
cadd n/Jane Doe p/91234567 e/jane@gmail.com a/Random Plac

The error looks like this:

However when changing the email, the client is added despite all other fields being the same; typing "cadd n/Jane Doe p/91234567 e/ane@gmail.com a/Random Place" adds the client:

There should be rules specified in the UG what constitutes a duplicate, I am unsure what the intended criteria is and hence am considering this a bug.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: EthanTheGoondu/ped#3

### PR DESCRIPTION
**Pull request by dhafinrazaq**
Fixes #183

### CODE REVIEW

**Comment by dhafinrazaq** [comment]
fixes #183

File: docs/UserGuide.md [hunk 1]
```diff
@@ -104,6 +104,14 @@ If you have a new client interested in your fitness training services, FitEgo ca
 
 All [client-related commands](#32-client-related-keywords) use prefix `c`. 
 
+<div markdown="block" class="alert alert-info"> 
+
+:information_source: **Note:**
+FitEgo will not allow you to have clients with the same email. This is because we believe that email is the most
+suitable identifier to differentiate clients.
```
Comment (dhafinrazaq) [inline docs/UserGuide.md]: is this too technical/revealing the implementation?

**Review by dhafinrazaq — COMMENTED** [review]
Question

File: docs/UserGuide.md (hunk 1 above)
Comment (benclmnt) [reply docs/UserGuide.md]: Perhaps it is better to just say "Each client should have a unique email"

**Review by benclmnt — APPROVED** [review]

**Comment by dhafinrazaq** [comment]
fixes #198

**Pull request merged**